source/scripts/tests/OrcTest.js

```diff
@@ -0,0 +1,41 @@
+import Expect from "expect.js"
+
+import Game from "scripts/model/Game.js"
+
+import MONSTERS from "scripts/data/monsters.js"
+
+import DATA from "scripts/data/index.js"
+
+export default function OrcTest() {
+
+    // We are a testing a game.
+
+    var game = new Game({
+
+        // A game with an adventurer.
+        adventurer: {
+            position: {x: 3, y: 3},
+        },
+
+        // A game with an adventurer and a bat.
+
+        monsters: [
+            {
+                position: {x: 1, y: 1},
+                protomonster: MONSTERS.RED_ORC
+            }
+        ]
+
+        // Also, by purposefully forgetting to
+        // pass a "wave" to the game, our test
+        // won't be interrupted by other monsters
+        // being spawned in.
+    })
+    Expect(game.monsters[0].position).to.eql({x: 1, y: 1})
+    game.monsters[0].onAction()
+    Expect(game.monsters[0].position).not.to.eql({x: 1, y: 1})
```

Comment:
Could we be a bit more specific here? Instead of asserting _not_ equal to 1x1, could we just assert 1x2 (or wherever the monster should move from here).